content/articles/payment-methods.markdown

```diff
@@ -7,20 +7,23 @@ categories:
 
 # Payment Methods
 
-DNSimple only supports two payment methods:
+DNSimple supports two payment methods:
 
 ## Credit card
 
 <note>
 DNSimple doesn't store credit card information. We use a PCI-compliant billing provider to process and store all payment data.
 </note>
 
-When you subscribe to DNSimple you will be required to provide a credit card. This card will be used to collect the payment for the subscription fees and all the purchases that you make in DNSimple. These purchases include:
+When you subscribe to DNSimple you will be required to provide a credit card. This card will be used to collect the payment for the subscription fees and all the purchases that you make in DNSimple. These purchases may include:
+
 - Domain registrations
 - Domain renewals
 - Domain transfers
+- Whois privacy
 - Comodo single-name SSL certificates
 - Comodo wildcard certificates
+- Email forwarding
```

Comment:
Are you sure we should add Email Forwarding? Technically it's not a purchase because it's not paid immediately. I've always considered part of the subscription fee.

Reply:
Fair enough, go ahead and remove it.